index.html

```diff
@@ -113,158 +114,7 @@ <h1>
-            </div>
-          </div>
-        </article>
-
-        <article>
-          <div class="placeholder"></div>
-          <div class="card_container1">
-            <h4>Multi-Post Stories Gain+Glory</h4>
-            <ul>
-              <div>
-                <li>Ruby on rails</li>
-              </div>
-              <div>
-                <li>css</li>
-              </div>
-              <div>
-                <li>Javscript</li>
-              </div>
-              <div>
-                <li>html</li>
-              </div>
-            </ul>
-            <div class="div_button">
-              <button class="btn_see_projects btn_effects" type="button" href="#">See Projects</button>
-            </div>
-          </div>
-        </article>
-
-        <article>
-          <div class="placeholder"></div>
-          <div class="card_container1">
-            <h4>Multi-Post Stories Gain+Glory</h4>
-            <ul>
-              <div>
-                <li>Ruby on rails</li>
-              </div>
-              <div>
-                <li>css</li>
-              </div>
-              <div>
-                <li>Javscript</li>
-              </div>
-              <div>
-                <li>html</li>
-              </div>
-            </ul>
-            <div class="div_button">
-              <button class="btn_see_projects btn_effects" type="button" href="#">See Projects</button>
-            </div>
-          </div>
-        </article>
-
-        <article>
-          <div class="placeholder"></div>
-          <div class="card_container1">
-            <h4>Multi-Post Stories Gain+Glory</h4>
-            <ul>
-              <div>
-                <li>Ruby on rails</li>
-              </div>
-              <div>
-                <li>css</li>
-              </div>
-              <div>
-                <li>Javscript</li>
-              </div>
-              <div>
-                <li>html</li>
-              </div>
-            </ul>
-            <div class="div_button">
-              <button class="btn_see_projects btn_effects" type="button" href="#">See Projects</button>
-            </div>
-          </div>
-        </article>
-
-        <article>
-          <div class="placeholder"></div>
-          <div class="card_container1">
-            <h4>Multi-Post Stories Gain+Glory</h4>
-            <ul>
-              <div>
-                <li>Ruby on rails</li>
-              </div>
-              <div>
-                <li>css</li>
-              </div>
-              <div>
-                <li>Javscript</li>
-              </div>
-              <div>
-                <li>html</li>
-              </div>
-            </ul>
-            <div class="div_button">
-              <button class="btn_see_projects btn_effects" type="button" href="#">See Projects</button>
-            </div>
-          </div>
-        </article>
-      </div>
-    </section>
+    <section class="section_works" id="Portoflio"></section>
```

Comment:
Please make sure the Lorem Ipsum text is not overflowing from the modal and there is padding on the bottom of the text.
<img width="822" alt="Screenshot 2023-01-06 at 1 30 55 AM" src="https://user-images.githubusercontent.com/25887253/210896509-48721965-19b3-4515-ba5e-11d8e0909017.png">
Figma template:
<img width="423" alt="Screenshot 2023-01-06 at 1 31 17 AM" src="https://user-images.githubusercontent.com/25887253/210896517-6ab77e5f-358a-4bfe-9a8a-e26abff1aa9c.png">